containers/rust-postgres/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.8'
+
+volumes:
+  postgres-data:
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Use the VARIANT arg to pick a Debian OS version: buster, bullseye
+        # Use bullseye when on local on arm64/Apple Silicon.
+        VARIANT:  bullseye
+        # Options
```

Comment:
from a quick glance, the 'base' Rust container doesn't include installing node.js, and since I don't think this one does either, so we can remove `NODE_VERSION` this from `args` -- thoughts?